src/Console/PrintSchemaCommand.php

```diff
@@ -24,6 +24,7 @@ class PrintSchemaCommand extends Command
 {--D|disk= : The disk to write the file to}
 {--json : Output JSON instead of GraphQL SDL}
 {--federation : Include federation directives and exclude federation spec additions, like _service.sdl}
+{--sort: Sort all types, fields, and arguments in alphabetical order, when possible}
```

Comment:
What do you mean `when possible`?

Reply:
I initially added that `when possible` before I had added the checks to throw on an invalid combination of flags, so it was intended to communicate that `--sort` is incompatible with `--json` and `--federation`.

I see how that could be confusing now, so I'll delete it.